curriculum/challenges/spanish/04-data-visualization/json-apis-and-ajax/convert-json-data-to-html.spanish.md

```diff
@@ -8,6 +8,7 @@ localeTitle: Convertir datos JSON a HTML
 
 ## Description
 <section id="description"> Ahora que está obteniendo datos de una API JSON, puede mostrarlos en el HTML. Puede usar un método <code>forEach</code> para recorrer los datos, ya que los objetos de la foto del gato se mantienen en una matriz. A medida que llegas a cada elemento, puedes modificar los elementos HTML. Primero, declare una variable html con <code>var html = &quot;&quot;;</code> . Luego, recorra el JSON, agregando HTML a la variable que envuelve los nombres de las claves en etiquetas <code>strong</code> , seguido del valor. Cuando el bucle termina, lo renderizas. Aquí está el código que hace esto: <blockquote> json.forEach (función (val) { <br> var keys = Object.keys (val); <br> html + = &quot;&lt;div class = &#39;cat&#39;&gt;&quot;; <br> keys.forEach (function (key) { <br> html + = &quot;&lt;strong&gt;&quot; + clave + &quot;&lt;/strong&gt;:&quot; + val [clave] + &quot;&lt;br&gt;&quot;; <br> }); <br> html + = &quot;&lt;/div&gt; &lt;br&gt;&quot;; <br> }); </blockquote></section>
+<b>Nota:</b> Para completar este reto es necesario añadir nuevos elementos html, por lo que no podrá depender de textContent. Deberá utilizar el método innerHTML que podría hacer que su página web sea vulnerable a ataques de cross site scripting.
```

Comment:
Something I did not catch in the first review, but according to our guidelines, notes are surrounded by `strong` tags.
```suggestion
<strong>Nota:</strong> Para completar este reto es necesario añadir nuevos elementos html, por lo que no podrá depender de textContent. Deberá utilizar el método innerHTML que podría hacer que su página web sea vulnerable a ataques de cross site scripting.
```